mesonbuild/modules/external_project.py

```diff
@@ -137,7 +137,7 @@ def _configure(self, state: 'ModuleState') -> None:
 
         if self.env.is_cross_build():
             host = '{}-{}-{}'.format(self.host_machine.cpu_family,
-                                     self.build_machine.system,
+                                     'unknown',
```

Comment:
I thought x86 machines were supposed to be `pc`?
so something like `'unknown' if self.host_machine.cpu_family != 'x86' else 'pc'`?

but also, I think the use of `self.host_machine.cpu_family` here looks wrong in some cases, like `x86', shouldn't that be some form of `i<x>86`?

Reply:
@xclaesse, you're the expert here

Reply:
Done.
Now I get `--host=i386-pc-linux`.
But this is of course a slightly more
radical change...